index.js

```diff
@@ -179,4 +183,8 @@ class EmailNotifier extends NotificationBase {
     }
 }
 
-module.exports = EmailNotifier;
+module.exports = {
+    EmailNotifier: EmailNotifier,
+    validateConfigEmail: validateConfigEmail
```

Comment:
can we just call this `validateConfig` Context is already implied the module this belongs to.